src/main/java/org/jmhsrobotics/frc2023/commands/AutoBalance.java

```diff
@@ -0,0 +1,52 @@
+package org.jmhsrobotics.frc2023.commands;
+
+//import org.apache.commons.io.filefilter.FalseFileFilter;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import org.jmhsrobotics.frc2023.RobotMath;
+import org.jmhsrobotics.frc2023.Constants.ArenaConstants;
+import org.jmhsrobotics.frc2023.Constants.AutoConstants;
+import org.jmhsrobotics.frc2023.RobotMath.DiminishingAverageHandler;
+import org.jmhsrobotics.frc2023.subsystems.Drivetrain;
+
+public class AutoBalance extends CommandBase {
+	private Drivetrain drivetrain;
+	private double moveLimit;
+	private DiminishingAverageHandler lowPassFilter;
+	private PIDController pidController;
```

Comment:
Why is the file inside of a commit just changing the pinout labels? Isn't there already an auto balance command?